robyn/__main__.py

```diff
@@ -77,10 +82,17 @@ def docs():
     webbrowser.open("https://sansyrox.github.io/robyn/#/")
```

Comment:
This particular link is deprecated. It should redirect to the Robyn hosted docs:  https://github.com/sparckles/docs-base. 

@sansyrox I will create a separate issue for this.

Reply:
Hey 👋🏻  @shivaylamba The new version of the [docs](https://github.com/sparckles/docs-base) is still in development so, that is why we are not  updating it in `robyn/__main__.py`. But the link is broken so we can update with these [docs](https://robyn.tech/)